src/sentry/static/sentry/app/utils/marked.tsx

```diff
@@ -0,0 +1,69 @@
+import marked from 'marked'; // eslint-disable-line no-restricted-imports
+import dompurify from 'dompurify';
+
+// Only https and mailto, (e.g. no javascript, vbscript, data protocols)
+const safeLinkPattern = /^(https?:|mailto:)/i;
+
+const safeImagePattern = /^https?:\/\/./i;
+
+function isSafeHref(href: string, pattern: RegExp) {
+  try {
+    return pattern.test(decodeURIComponent(unescape(href)));
+  } catch (e) {
+    return false;
+  }
+}
+
+// We need to implement our own marked Renderer to not render potentially
+// malicious uris.
+
+class SafeRenderer extends marked.Renderer {
+  link(href: string, title: string, text: string) {
+    // For a bad link, just return the plain text href
+    if (!isSafeHref(href, safeLinkPattern)) {
+      return href;
+    }
+
+    let out = '<a href="' + href + '"';
+    if (title) {
+      out += ' title="' + title + '"';
+    }
+    out += '>' + text + '</a>';
+
+    return dompurify.sanitize(out);
+  }
+
+  image(href: string, title: string, text: string) {
+    // For a bad image, return an empty string
+    if (this.options.sanitize && !isSafeHref(href, safeImagePattern)) {
+      return '';
+    }
+
+    let out = '<img src="' + href + '" alt="' + text + '"';
+    if (title) {
+      out += ' title="' + title + '"';
+    }
+    out += this.options.xhtml ? '/>' : '>';
```

Comment:
Another place you could use template literals. Also our application is not xhtml so you could remove the related option.